nemo_skills/dataset/bfcl/__init__.py

```diff
@@ -0,0 +1,41 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
should we call the folder and eval group bfcl-v3 to avoid confusion with other versions?